pydis_site/apps/home/templatetags/wiki_extra.py

```diff
@@ -0,0 +1,75 @@
+from typing import Any, Dict, Type, Union
+
+from django import template
+from django.forms import BooleanField, BoundField, CharField, Field, ImageField, ModelChoiceField
+from django.template import Context
+from django.template.loader import get_template
+from django.utils.safestring import mark_safe
+from wiki.editors.markitup import MarkItUpWidget
+from wiki.forms import WikiSlugField
+from wiki.models import URLPath
+from wiki.plugins.notifications.forms import SettingsModelChoiceField
+
+TEMPLATE_PATH = "wiki/forms/fields/{0}.html"
+
+TEMPLATES: Dict[Type, str] = {
+    BooleanField: TEMPLATE_PATH.format("boolean"),
+    CharField: TEMPLATE_PATH.format("char"),
+    ImageField: TEMPLATE_PATH.format("image"),
+
+    ModelChoiceField: TEMPLATE_PATH.format("model_choice"),
+    SettingsModelChoiceField: TEMPLATE_PATH.format("model_choice"),
+    WikiSlugField: TEMPLATE_PATH.format("wiki_slug_render"),
+}
+
+
+register = template.Library()
+
+
+def get_unbound_field(field: BoundField) -> Field:
+    while isinstance(field, BoundField):
+        field = field.field
+
+    return field
+
+
+def render(template_path: str, context: Dict[str, Any]):
+    return mark_safe(get_template(template_path).render(context))  # noqa: S703 S308
+
+
+@register.simple_tag
+def render_field(field: Field, render_labels: bool = True):
+    if isinstance(field, BoundField):
+        unbound_field = get_unbound_field(field)
+    else:
+        unbound_field = field
```

Comment:
do you need the extra `isinstance` check here? I feel like this entire block could just be
```py
unbound_field = get_unbound_field(field)
```
instead.